app/services/word-service.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+import PastWord from 'lyricsss/models/past-word';
+
+export default Ember.Service.extend({
+  add(word, team, answerCorrect, time) {
```

Comment:
Why is this desirable? With that, when calling the function you have to add an object key to each argument in addition to argument value. Improves readability but makes the argument twice as complicated.